infrastructure/k8s/helm-chart/eoapi/templates/db/manifest.yaml

```diff
@@ -58,14 +61,14 @@ spec:
             - "-N"
             - "500"
           env:
-            # NOTE: there's no reason we need to use a `ConfigMap` here to get os env vars into the pod.
+            # NOTE: there's no reason we need to use a `ConfigMap` or `Secret` here to get os env vars into the pod.
             # we could just template them out here immediately with `value: $_` but this allows us
             # to store them in k8s intermediately and change them and then bounce deploys if needed
-            {{- range $envKey, $_ := .Values.db.settings.envVars }}
+            {{- range $envKey, $_ := .Values.db.settings.secrets }}
             - name: {{ upper $envKey }}
               valueFrom:
-                configMapKeyRef:
-                  name: pgstac-env-configmap
+                secretKeyRef:
```

Comment:
comment for reviewer: just like a `configMapKeyRef` but for secrets and it does all the decoding for us